rundeckapp/grails-app/assets/javascripts/application.js

```diff
@@ -383,52 +383,13 @@ function _setupAceTextareaEditor(textarea, callback, autoCompleter) {
 
   //create editor
   var editor = ace.edit(generateId(_shadow));
-  editor.$blockScrolling = Infinity;
-
-  var checkResize;
-  if (data.aceResizeMax) {
-    var heightPx = parseInt(height.replace(/px$/, ''));
-    var lineheight = editor.renderer.lineHeight;
-    var checkSize = Math.floor(heightPx / lineheight);
-    var checkSizeMax = parseInt(data.aceResizeMax);
-    if (checkSize < checkSizeMax) {
-
-      var checkSizeInc = 5;
-      var pxInc = lineheight;
-      checkResize = function (editor) {
-        "use strict";
-        var lineCount = editor.session.getLength();
-        if (lineCount > checkSize && checkSize < checkSizeMax) {
-          var diff = Math.min(checkSizeMax - checkSize, lineCount - checkSize);
-          var increment = Math.max(checkSizeInc, diff);
-          checkSize += increment;
-          heightPx += increment * pxInc;
-          _shadow.css({
-            height: heightPx + 'px'
-          });
-          editor.resize();
-        }
-      };
-
-      if (data.aceResizeAuto) {
-        checkResize(editor);
-      }
-    }
-  }
+  editor.setOptions({
+    maxLines: Infinity,
+    minLines: 10
+  });
 
   setAceSyntaxMode(data.aceSessionMode, editor);
   editor.setTheme("ace/theme/chrome");
-  editor.getSession().on('change', function (e) {
```

Comment:
Is this entire event handler no longer necessary?

Reply:
It seems to only be used to trigger the resize action.
I've tested it without (only using Chrome so far) and I don't see any difference in behaviour so far.

Reply:
I believe the `callback` at least still needs to be triggered by change events, it is used on other pages which setup ace textareas.  I'm not sure about the "synthetic change" part of this, I think @ProTip you were the one who added that code here https://github.com/rundeck/rundeck/commit/95d84adcbfd493538d11108bbf96d0b976718f7a

Reply:
I'd suggest keeping the old behavior, and simply remove the resize part if it is no longer necessary

Reply:
@gschueler Sorry for getting back to this so late; the synthetic event was added so plugins could listen to changes more easily. Removing just resize if it is no longer needed is probably the way.